Cargo.toml

```diff
@@ -14,8 +14,8 @@ license = "MIT OR Apache-2.0"
 [dependencies]
 bdk-macros = "^0.6"
 log = "^0.4"
-miniscript = { version = "9.0", default-features = false, features = ["serde"] }
```

Comment:
This should work but will still force any downstream projects that use `bdk` and have `default-features = false` to now have to also enable a new `std` feature. That's the kind of breaking change I want to avoid on a patch release. Ideally I want to defer all breaking changes to the 1.0 release. 

Besides Mutiny are there any other downstream projects that need this change?